### ISSUE TITLE
valid_product_attribute_value_wnva_ids does not exist

### ISSUE THREAD

**Issue by stesifrancesco** [odoo/odoo#48473]
Impacted versions:
Odoo 12 EE (I think also CE but not tested)

Steps to reproduce:
Upgrade Odoo to latest 12.0 commit

Current behavior:
When I install MRP i got the error:
valid_product_attribute_value_wnva_ids does not exist

Expected behavior:
I want ust install MRP

Video/Screenshot link (optional):
https://youtu.be/0Vj-iCpggW4

**Comment by pedrobaeza** [comment]
Have you updated all your modules and restart service?

**Comment by stesifrancesco** [comment]
Good Morning.
Of course !
I've just update my jit repository (Git pull both on EE and CE) ..... restarted my service, created a new empty DB and like in the video above I've tried to install MRP

Thanks for your help

**Comment by stesifrancesco** [comment]
This is the full error:

`Error:
Odoo Server Error

Traceback (most recent call last):
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 758, in parse
    self._tags[rec.tag](rec, de, mode=mode)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 663, in _tag_record
    record = model.with_context(rec_context)._load_records([data], self.mode == 'update')
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3888, in _load_records
    records = self._load_records_create([data['values'] for data in to_create])
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3802, in _load_records_create
    return self.create(values)
  File "<decorator-gen-175>", line 2, in create
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 443, in _model_create_single
    return self.browse().concat(*(create(self, vals) for vals in arg))
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 443, in <genexpr>
    return self.browse().concat(*(create(self, vals) for vals in arg))
  File "/home/odoo/odoo-dev12/odoo/addons/stock/models/stock_warehouse.py", line 103, in create
    warehouse = super(Warehouse, self).create(vals)
  File "<decorator-gen-3>", line 2, in create
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 461, in _model_create_multi
    return create(self, [arg])
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3583, in create
    records = self._create(data_list)
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3683, in _create
    cr.execute(query, params)
  File "/home/odoo/odoo-dev12/odoo/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.IntegrityError: duplicate key value violates unique constraint "stock_warehouse_warehouse_name_uniq"
DETAIL:  Key (name, company_id)=(My Company, 1) already exists.


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 656, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 314, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 698, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 346, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/home/odoo/odoo-dev12/odoo/odoo/http.py", line 519, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/odoo-dev12/odoo/addons/web/controllers/main.py", line 966, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo/odoo-dev12/odoo/addons/web/controllers/main.py", line 954, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 759, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 746, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<decorator-gen-61>", line 2, in button_immediate_install
  File "/home/odoo/odoo-dev12/odoo/odoo/addons/base/models/ir_module.py", line 74, in check_and_log
    return method(self, *args, **kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/addons/base/models/ir_module.py", line 445, in button_immediate_install
    return self._button_immediate_function(type(self).button_install)
  File "/home/odoo/odoo-dev12/odoo/odoo/addons/base/models/ir_module.py", line 561, in _button_immediate_function
    modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "/home/odoo/odoo-dev12/odoo/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/home/odoo/odoo-dev12/odoo/odoo/modules/loading.py", line 421, in load_modules
    loaded_modules, update_module, models_to_check)
  File "/home/odoo/odoo-dev12/odoo/odoo/modules/loading.py", line 313, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/home/odoo/odoo-dev12/odoo/odoo/modules/loading.py", line 222, in load_module_graph
    load_data(cr, idref, mode, kind='data', package=package, report=report)
  File "/home/odoo/odoo-dev12/odoo/odoo/modules/loading.py", line 68, in load_data
    tools.convert_file(cr, package.name, filename, idref, mode, noupdate, kind, report)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 802, in convert_file
    convert_xml_import(cr, module, fp, idref, mode, noupdate, report)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 865, in convert_xml_import
    obj.parse(doc.getroot(), mode=mode)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 755, in parse
    self.parse(rec, mode)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 764, in parse
    exc_info[2]
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/pycompat.py", line 86, in reraise
    raise value.with_traceback(tb)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 758, in parse
    self._tags[rec.tag](rec, de, mode=mode)
  File "/home/odoo/odoo-dev12/odoo/odoo/tools/convert.py", line 663, in _tag_record
    record = model.with_context(rec_context)._load_records([data], self.mode == 'update')
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3888, in _load_records
    records = self._load_records_create([data['values'] for data in to_create])
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3802, in _load_records_create
    return self.create(values)
  File "<decorator-gen-175>", line 2, in create
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 443, in _model_create_single
    return self.browse().concat(*(create(self, vals) for vals in arg))
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 443, in <genexpr>
    return self.browse().concat(*(create(self, vals) for vals in arg))
  File "/home/odoo/odoo-dev12/odoo/addons/stock/models/stock_warehouse.py", line 103, in create
    warehouse = super(Warehouse, self).create(vals)
  File "<decorator-gen-3>", line 2, in create
  File "/home/odoo/odoo-dev12/odoo/odoo/api.py", line 461, in _model_create_multi
    return create(self, [arg])
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3583, in create
    records = self._create(data_list)
  File "/home/odoo/odoo-dev12/odoo/odoo/models.py", line 3683, in _create
    cr.execute(query, params)
  File "/home/odoo/odoo-dev12/odoo/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/home/odoo/odoo-dev12/odoo/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
odoo.tools.convert.ParseError: "duplicate key value violates unique constraint "stock_warehouse_warehouse_name_uniq"
DETAIL:  Key (name, company_id)=(My Company, 1) already exists.
" while parsing /home/odoo/odoo-dev12/odoo/addons/stock/data/stock_data.xml:125, near
<record id="warehouse0" model="stock.warehouse">
            <field name="partner_id" ref="base.main_partner"/>
            <field name="code">WH</field>
        </record>`

**Comment by pedrobaeza** [comment]
But that's another error that the warehouse name is not unique, so you are doing something incorrect on your end.

Are you able to reproduce the problem on runbot (https://runbot.odoo.com - admin/admin)?

**Comment by stesifrancesco** [comment]
Sorry and .... Yes you are right ! I got the different error after a new Git Pull from Odoo CC ....where I got 1 new file.....
Anyway..... The error was on a database created with the same name as a newly deleted DB.
I deleted the DB again and recreated it with a different name.
Now it seems to work properly .... maybe there is something left in the browser cache that can create problems or in the filestore?

**Comment by stesifrancesco** [comment]
Yes, sorry, you're right, I made a new PULL from Git CE and immediately after or tried the installation again, getting an error that I copied thinking it was the same. Now I have deleted the DB again and recreated a DB with a different name and the installation was successful.
The error was on a database created with the same name as a newly deleted DB.
I deleted the DB again and recreated it with a different name.
Now it seems to work properly .... maybe there is something left in the browser cache that can create problems or in the filestore?
